src/main/kotlin/org/usecase/dao/HealthCheckDao.kt

```diff
@@ -0,0 +1,19 @@
+package org.usecase.dao
```

Comment:
The problem of using this path is that all content inside it is created dynamically. So, if there is a table named health_check, its DAO will conflict with this file.